.gitignore

```diff
@@ -40,6 +40,7 @@ website/i18n/**/*
 #!website/i18n/fr/**/*
 test-appsmith-docusaurus/.env
 .vale.ini
+vale.ini
```

Comment:
vale.ini is not the file it's .vale.ini only. Do we need this change?

Reply:
For me and Dancia it was vale.ini , that's why I added it. Linter works fine though